### PR TITLE
Various placeholders constants etc

### PR DESCRIPTION
**Pull request by balghane**
Guesstimates from physics on the smart motion profile constraints and some other fixes and things

### CODE REVIEW

File: src/main/java/frc/robot/Cal.java [hunk 1]
```diff
@@ -69,30 +73,35 @@ public static final class Intake {
...
+    public static final double CLAMP_POSITION_THRESHOLD_DEGREES = 110.0;
 
     /** Absolute encoder position when the arm is at 0 degrees */
     public static final double ABSOLUTE_ENCODER_START_POS_DEG = Cal.PLACEHOLDER_DOUBLE;
 
     /** Voltage required to hold the intake in the horizontal position */
-    public static final double ARBITRARY_FEED_FORWARD_VOLTS = Cal.PLACEHOLDER_DOUBLE;
+    // Stall torque: 3.36 Nm * 75 = 252 Nm
+    // Max torque: 6 kg * 9.81 (N/kg) * 0.26m = 15.3 Nm
+    // Ratio: 0.73 V
```
Comment (tails618) [inline src/main/java/frc/robot/Cal.java:93]: Should these be /** */ comments? (Will there be any reason to want to see this from somewhere else, or is it just for reference should we need it?)

E: applies also to similar comments, like lines 98-99

File: src/main/java/frc/robot/commands/AutoScoreAndBalance.java [hunk 2]
```diff
@@ -14,9 +14,12 @@ public class AutoScoreAndBalance extends SequentialCommandGroup {
   private PathPlannerTrajectory traj =
       PathPlanner.loadPath(
           "ScoreAndBalanceTraj",
-          new PathConstraints(Cal.PLACEHOLDER_DOUBLE, Cal.PLACEHOLDER_DOUBLE));
+          new PathConstraints(
+              Cal.SwerveSubsystem.MAX_LINEAR_SPEED_METERS_PER_SEC,
+              Cal.SwerveSubsystem.MAX_LINEAR_ACCELERATION_METERS_PER_SEC_SQ));
 
   public AutoScoreAndBalance(boolean isFirstPath, Lift lift, DriveSubsystem drive) {
+    // TODO update this since scoreGamePiece and doneScoring are stubbed
```
Comment (tails618) [inline src/main/java/frc/robot/commands/AutoScoreAndBalance.java:22]: VS Code doesn't underline `// TODO` in blue, only `//TODO`. Idk if that matters, though.

File: src/main/java/frc/robot/subsystems/Lift.java [hunk 3]
```diff
@@ -431,8 +427,7 @@ public void periodic() {
    * move as soon as the lift is clear of that zone
    */
   public boolean clearOfIntakeZone() {
-    if ((elevatorLeftEncoder.getPosition() > Cal.Lift.ELEVATOR_INTAKE_ZONE_THRESHOLD_INCHES)
-        || (armEncoder.getPosition() > Cal.Lift.ARM_INTAKE_ZONE_THRESHOLD_DEGREES)) {
+    if (armEncoder.getPosition() > Cal.Lift.ARM_INTAKE_ZONE_THRESHOLD_DEGREES) {
```
Comment (tails618) [inline src/main/java/frc/robot/subsystems/Lift.java]: Just return this value instead of having a conditional.

**Review by tails618 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Cal.java (hunk 1 above)
Comment (balghane) [reply src/main/java/frc/robot/Cal.java:93]: nah they're just for reference, and I checked that the /**/ comments are still the hover-text

File: src/main/java/frc/robot/commands/AutoScoreAndBalance.java (hunk 2 above)
Comment (balghane) [reply src/main/java/frc/robot/commands/AutoScoreAndBalance.java:22]: IDK I've been keeping track of our TODOs just by searching them, we do still have a number of them to burn down

File: src/main/java/frc/robot/subsystems/Lift.java (hunk 3 above)
Comment (balghane) [reply src/main/java/frc/robot/subsystems/Lift.java]: Good call

**Review by tails618 — APPROVED** [review]

**Pull request merged**